cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -559,12 +523,255 @@ def add_reimbursement_taxable_new_doc(self):
+            hra_component=get_company.hra_component
+
+            ss_assignment = frappe.get_list(
+                'Salary Structure Assignment',
+                filters={'employee': self.employee, 'docstatus': 1,'company':self.company,"custom_payroll_period":self.custom_payroll_period},
+                fields=['name','from_date','custom_payroll_period','salary_structure'],
+                order_by='from_date desc',
+            )
+
+            if ss_assignment:
+                first_assignment = next(iter(ss_assignment))  
+                first_assignment_date = first_assignment.get("from_date")
```

Ruff (0.8.2)

Local variable `first_assignment_date` is assigned to but never used

Remove assignment to unused variable `first_assignment_date`

(F841)